.gitignore

```diff
@@ -43,6 +43,7 @@ validation-staus.json
 .CVS
 .idea
 node_modules
+vendor
```

Comment:
Is PR 1746 required to be approved to enable this branch to go in?

Appreciate that we could just revert this if needed.

Reply:
That PR has gone in now